### PR TITLE
docs: add enum values

### PR DESCRIPTION
**Pull request by byashimov**
Adds enum values to the docs.

### CODE REVIEW

**Comment by byashimov** [comment]
[tests](https://github.com/aiven/terraform-provider-aiven/actions/runs/9188760711)

File: docs/resources/service_integration_endpoint.md [hunk 1]
```diff
@@ -137,17 +137,17 @@ Required:
...
 
-- `sasl_mechanism` (String) SASL mechanism used for connections to the Kafka server.
+- `sasl_mechanism` (String) Enum: `PLAIN`, `SCRAM-SHA-256`, `SCRAM-SHA-512`. SASL mechanism used for connections to the Kafka server.
 - `sasl_plain_password` (String, Sensitive) Password for SASL PLAIN mechanism in the Kafka server.
 - `sasl_plain_username` (String) Username for SASL PLAIN mechanism in the Kafka server.
 - `ssl_ca_cert` (String) PEM-encoded CA certificate.
 - `ssl_client_cert` (String) PEM-encoded client certificate.
 - `ssl_client_key` (String) PEM-encoded client key.
-- `ssl_endpoint_identification_algorithm` (String) The endpoint identification algorithm to validate server hostname using server certificate.
+- `ssl_endpoint_identification_algorithm` (String) Enum: `https`, ``. The endpoint identification algorithm to validate server hostname using server certificate.
```
Comment (rriski) [inline docs/resources/service_integration_endpoint.md:150]: 🔎 This should be fixed in go-api-schemas https://github.com/aiven/go-api-schemas/blob/25035ec40a7db6153c6e992f86765de7a8064a0d/pkg/dist/integration_endpoint_types.yml#L311. Can be done later.

File: internal/sdkprovider/userconfig/serviceintegration/logs.go [hunk 2]
```diff
@@ -27,7 +27,7 @@ func logsUserConfig() *schema.Schema {
 			"selected_log_fields": {
 				Description: "The list of logging fields that will be sent to the integration logging service. The MESSAGE and timestamp fields are always sent.",
 				Elem: &schema.Schema{
-					Description:  "Log field name.",
+					Description:  "Enum: `HOSTNAME`, `PRIORITY`, `REALTIME_TIMESTAMP`, `service_name`, `SYSTEMD_UNIT`. Log field name.",
```
Comment (rriski) [inline internal/sdkprovider/userconfig/serviceintegration/logs.go:30]: 🔎 `service_name` looks strange here but it seems to be valid.

**Review by rriski — APPROVED** [review]
LGTM

**Pull request merged**